techlottus/portalverse-uane
# [DEV-5266] Migrate "cintillo" component

## Issue
 - Migrate the new "cintillo" component from UTEG repo to that repo

## Solution
 - [feat: Cintillo component migrated](https://github.com/techlottus/portalverse-uane/pull/148/commits/01cbfc53c843650b2569f3aefaa0e1884a301931)

## Testing
 - Go to `/egresados`
 - Search this banner 
![image](https://user-images.githubusercontent.com/62124025/234422274-33396d2b-835d-4dec-80ad-df9dbb4d6fcd.png)
 - Change size of the browser and check that the image is not distorted
